### PR TITLE
Export method

### PR DESCRIPTION
**Pull request by Lemorra**
Added support for exports as .tiff, .csv in the 'Save As' Window
tiffSave2
csvSave

### CODE REVIEW

File: utils.py [hunk 1]
```diff
@@ -211,62 +211,109 @@ def saveImage(window):
...
     if file:
-        filename = file.name
-
-        if (
-            messagebox.askquestion(
-                "Save options", "Do you want to save with the annotations?"
+        filename = file.split("/")[-1]
+
+        if filename.endswith('.tiff'):
+            if (messagebox.askquestion("Before we proceed",f"Are you sure you want to export image as {filename}?") == "yes"):
```
Comment (rsnk96) [inline utils.py:222]: @Lemorra - can you mention in this message that the TIFF will be saved with Kelvin values, and not a false colour mapping?

File: utils.py [hunk 2]
```diff
@@ -211,62 +211,109 @@ def saveImage(window):
...
+                tiffExport(window, file)
+            else:
+                messagebox.showinfo("Export Cancelled", f"You have cancelled the process of exporting {filename}")
+
+        elif filename.endswith('.csv'):
+            if (messagebox.askquestion("Before we proceed",f"Are you sure you want to export image as {filename}?") == "yes"):
```
Comment (rsnk96) [inline utils.py:228]: @Lemorra - can you mention in this message that the CSV will be saved with Kelvin values, and not a false colour mapping?

File: utils.py [hunk 3]
```diff
@@ -211,62 +211,109 @@ def saveImage(window):
...
+
+        else:
+            print(file)
+            pngExport(window, file, imageSurface, overlays, exthandler)
+
+def pngExport(window, filename, imageSurface, overlays, exthandler):
+    """Function to export image as .PNG format
+
+    Args:
+        window (class Window object): object that refers to main window
```
Comment (rsnk96) [inline utils.py:241]: You needn't have "class Window object", you can just mention <library>.Window :)

**Review by rsnk96 — CHANGES_REQUESTED** [review]

File: utils.py (hunk 1 above)
Comment (Lemorra) [reply utils.py]: Done.

File: utils.py (hunk 2 above)
Comment (Lemorra) [reply utils.py]: Done but please do check.

File: utils.py (hunk 3 above)
Comment (Lemorra) [reply utils.py]: Done.

**Pull request merged**